__tests__/constants/default-processor-options.ts

```diff
@@ -46,5 +46,8 @@ export const DefaultProcessorOptions: IIssuesProcessorOptions = Object.freeze({
   exemptAllPrAssignees: undefined,
   enableStatistics: true,
   labelsToRemoveWhenUnstale: '',
-  labelsToAddWhenUnstale: ''
+  labelsToAddWhenUnstale: '',
+  updatesResetStale: true,
```

Comment:
Based on your other PR which splits updates and comments. I wonder if we need a better catch-all term for this. What I don't want to do is to add 3 for motions for `updatesResetStale`, `commentsResetStale` and their issue/PR variants.

Reply:
maybe `activity`? 🤔

Reply:
9df6ee510eab7eb5dffb23726907f2799f4e3a06